src/components/DataInput/DropdownListInput.tsx

```diff
@@ -0,0 +1,35 @@
+"use client";
+
+import React, { ReactNode } from "react";
+import { MenuItem, InputLabel, Select, FormControl, SelectChangeEvent } from "@mui/material";
+
+interface Props {
+    optionLabels: string[];
+    optionValues: string[];
```

Comment:
for this sort of API, a [string,string][] type for options probably makes more sense as `optionLabels` and `optionValues` have a one-to-one correspondence